src/district/ui/smart_contracts/utils.clj

```diff
@@ -0,0 +1,34 @@
+(ns district.ui.smart-contracts.utils
+  (:require [clojure.string :as str]
+           [clojure.data.json :as json]))
+
+(defn get-abi-from-truffle-art [json-path]
+  (-> (slurp json-path)
+      (json/read-str :key-fn keyword)
+      :abi))
+
+(defmacro slurp-env-contracts []
+  (let [smart-contracts-file (System/getenv "SMART_CONTRACTS")
```

Comment:
I don't think you have that at macroexpansion time, I think those are things for the closure compiler, who runs after all this stuff

Reply:
ok, we can add them to the build file using some plugin, like https://github.com/athos/lein-with-env-vars

Reply:
Ok so according to this: https://clojurescript.org/reference/compiler-options#aot-cache
We shouldn't be using env variables because the build can be cached and cause hard-to-find errors, with old JS beinf=g served form the global cache. It is better to utilize `:closure-defines`

Reply:
Moved constants to closure-defines

Reply:
Moved back to env variables since couldn't make it work with closure-defines. I can't goog-define in clj file since it expands to a code that is only meant  to be inside cljs file